### PR TITLE
Update scripts: id, imgurl, cover for download covers.

### PR DESCRIPTION
**Pull request by lipebol**
Hello,

I'm new to programming and I've never programmed in shell script before.
I tried hard to try to make it as elegant as possible!

I believe Spotify changed the 'Metadata' structure, and the scripts didn't download the covers.
I changed the way it captures the 'id' and the 'imgurl'. 'imgurl' now comes under 'Metadata'.
In the 'cover' script I added a condition for it to check the 'mpris:spotify' and if it is disabled it writes 'id' = " " and the cover is 'empty.jpg'. I also changed the way it checks if the cover has already been downloaded.

### CODE REVIEW

File: scripts/imgurl.sh
Comment (Madh93) [inline scripts/imgurl.sh:1]: It's better to delete commented-out lines of code :smiley:

**Review by Madh93 — CHANGES_REQUESTED** [review]
Nice contribution! It works great.

File: scripts/imgurl.sh
Comment (lipebol) [reply scripts/imgurl.sh:1]: Concluded!
I read pull #18 (Update timestamp) and re-added the line.

**Review by Madh93 — APPROVED** [review]
Thanks! :1st_place_medal:

**Pull request merged**